CODING_GUIDELINES.md

```diff
@@ -43,11 +43,11 @@ if (condition) {
     //code
 }
 
-for(int a=0; a<b; ++b) {
+for (int a = 0; a < b; ++b) {
```

Comment:
This does not conflict.
>One exception could be: for loops.
This is RULE, and later - EXCEPTION. IMO, it is logical.